requirements.txt

```diff
@@ -5,7 +5,7 @@ click==7.1.2
 cycler==0.10.0
 decorator==4.4.2
 esse==2020.10.19.3
-exabyte-api-client==2020.10.19
```

Comment:
Let's make sure to make a matching change in requirements-dev.txt

Reply:
Bumped the version up